docs/_includes/footer.html

```diff
@@ -1,21 +1,21 @@
-  <div class="footer">
-    <div class="footer-links">
-      {% include links.html %}
-    </div>
-    <div class="copyright">
-      Copyright (c) 2016, Docker Inc.
-    </div>
-  </div>
-  <script src="/style/js/jquery-2.1.4.min.js"></script>
-  <script src="/style/js/highlight.min.js"></script>
-  <script src="/style/js/main.js"></script>
-  <script>
-    (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-    (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-    m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-    })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
-    ga('create', 'UA-71407002-1', 'auto');
-    ga('send', 'pageview');
-  </script>  
-  </body>
-</html>
+  <div class="footer">
+    <div class="footer-links">
+      {% include links.html %}
+    </div>
+    <div class="copyright">
+      Copyright (c) 2017 the containerd authors
```

Comment:
I might be wrong (or it may not matter in reality) but I think the copyright needs to span the whole time, not just the current year, so `2016-2017` would be more correct here (assuming the `2016` before was correct).